File: kotlin/struts/src/main/java/sample/kotlin/struts/HelloForm.java
package sample.kotlin.struts;

import org.apache.struts.action.ActionForm;

public class HelloForm extends ActionForm {
    private static final long serialVersionUID = 1L;
    
    private String message;

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }
    
    
}

File: kotlin/struts/src/main/kotlin/sample/kotlin/struts/HelloForm.kt
package sample.kotlin.struts

import org.apache.struts.action.ActionForm

class HelloForm: ActionForm() {
    private var message : String? = null
    
    fun getMessage() : String? {
        return this.message
    }
    
    fun setMessage(message : String?) {
        this.message = message
    }
}